services/usernote-helper.js

```diff
@@ -69,13 +78,21 @@ module.exports = {
           link: removedNote.l,
           index,
           t: removedNote.t,
-          timestamp: removedNote.t
+          timestamp: removedNote.t,
+          fromChannel: fromChannel
         };
       });
     });
   }
 };
 
+function checkAccess ({channel, subreddit}) {
+  if (!moduleConfig || !moduleConfig.channelPermissions || moduleConfig.channelPermissions[channel] && (_.isBoolean(moduleConfig.channelPermissions[channel]) || subreddit!==undefined && moduleConfig.channelPermissions[channel].findIndex(sub => sub.toLowerCase() === subreddit.toLowerCase()) > -1)) {
```

Comment:
This line is very hard to read. I think it would be better if it was split into something like

```js
if (!moduleConfig || !moduleConfig.channelPermissions) {
  return true;
}
if (_.isBoolean(moduleConfig.channelPermissions[channel])) {
}
// ...
```

Also, what happens if someone sets a value of `false` in channelPermissions?